src/table.ts

```diff
@@ -433,14 +433,14 @@ export class Column implements Ydb.Table.IColumnMeta {
     constructor(public name: string, public type: IType) {}
 }
 
-export class StorageSettings implements Ydb.Table.IStorageSettings {
-    constructor(public storageKind: string) {}
+export class StoragePool implements Ydb.Table.IStoragePool {
```

Comment:
What is the purpose of this renaming and what is a general advice to people who are already using StorageSettings class in their code :)?